src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px auto;
+  gap: $size-of-gap;
+  width: 7*$size-of-cell+6*$size-of-gap;
```

Comment:
What are 7 and 6? Would be great to make it a variable too or even calculate width separately